static/src/stylesheets/layout/new-header/_feedback-form.scss

```diff
@@ -0,0 +1,44 @@
+/*
+ * Temporary form for feedback on the desktop navigation
+ */
+
+.feedback-form {
```

Comment:
This needs some cross browser testing, because it will break in non flexbox browsers. Can take care of it today, when I work on the other issues though.